common/transport/http/src/rest_api_client.ts

```diff
@@ -273,7 +274,8 @@ export class RestApiClient {
 
 export namespace RestApiClient {
     export interface TransportConfig {
-        host: string;
+        host?: string;
+        socketPath?: string;
```

Comment:
isn't that supposed to be `host?: string | { socketPath: string }` and no `socketPath` property?

Reply:
That makes sense.